pypesto/sampling/parallel_tempering.py

```diff
@@ -61,7 +64,7 @@ def initialize(self,
     def sample(
             self, n_samples: int, beta: float = 1.):
         # loop over iterations
-        for i_sample in tqdm(range(int(n_samples))):
+        for i_sample in tqdm(range(int(n_samples))):  # TODO test
```

Comment:
TODO

Reply:
FYI: This was basically to also enable the switch-off of this progress bar.